pkg/pillar/hypervisor/kvm.go

```diff
@@ -799,7 +799,12 @@ func (ctx KvmContext) Setup(status types.DomainStatus, config types.DomainConfig
 	// check if vTPM is enabled
 	swtpmCtrlSock := ""
 	if status.VirtualTPM {
-		swtpmCtrlSock = fmt.Sprintf(types.SwtpmCtrlSocketPath, domainName)
+		domainUUID, _, _, err := types.DomainnameToUUID(domainName)
```

Comment:
@OhmSpectator this is the part I missed to update in the previous version, prevent VM from running becuse SWTPM socket is invalid.